### ISSUE TITLE
9 CHIRON-Eye For Blind

### ISSUE THREAD

**Issue by soumyajit4419** [CodeBreakMIT/CodeBreak1.0#12]
**Before you start, please follow this format for your issue title**:  
TEAM NUMBER TEAM NAME-PROJECT NAME

## ℹ️ Project information

- **Project Name**: Eye For Blind
- **Short Project Description**: 
- **Team ID**: 9
- **Team Name**: CHIRON
- **Team Members**:  1.SOUMYAJIT BEHERA  (https://github.com/soumyajit4419)
2.RAHUL KUMAR PATRO (https://github.com/Rahul1582)
3.AKSHAT DUBEY (https://github.com/dubeyakshat07)
4.ATMARAM PATRA (https://github.com/Atmaram07)

- **Repository Link**: https://github.com/soumyajit4419/Code_Break-Chiron-

- **Labels**:  Machine Learning, Web Development

## 🔥 Your Pitch
         EYE FOR BLIND               

 Its quite easy for us to see things happening around us or go to our places of interest be it a restaurant, park etc or identify our friends and family members. But what about those who can't see? So, here is VISION a web app for the blind. The app has three features -
Identifying faces - Its quite easy for us to spot our mom, dad or friends but for them this can be quite challenging. If the user finds all unknown people around him he can somewhat be alerted. The app is customized for every user. It identifies known and unknown faces and also tells the name of the known faces.
Describing what's going on around the user - This can help the blind in knowing if some harmful activities are going on around him and he should be away from such a place. Not necessarily harmful activities, for every single thing happening around these people, they have to rely on others to know about it. So, this feature can really help the blind in becoming more self-dependent.
Reading texts - The app can help the user by prompting small texts such as under construction board, alert or names of shop. Without reading certain warning boards a blind can really run into trouble.It also helps him to identify who is known to him and who is a stranger.
The app is completely voice automated so the user has no trouble using the app. The user says the command and the app automatically capture a picture and tells him whatever he wants to know. 

## Technology Stack Used
MACHINE LEARNING
OPEN CV
FLASK
JAVA SCRIPT
KERAS
TENSORFLOW
TESSERACT OCR

**Comment by soumyajit4419** [comment]
TEAM CHIRON ISSUE
